cal/cal/helpers.py

```diff
@@ -66,7 +66,7 @@ def handle_time_string(time_str, timezone_str):
     return time
 
 
-def get_time_series(model, timezone='UTC', time_step='weekly', calendar_ids=None, start=None, end=None):
+def get_time_series(model, timezone='UTC', time_step='Week', calendar_ids=None, start=None, end=None):
```

Comment:
I'm getting these parameters from the url, so when a request is for "eventWeek"/"eventDay"/"eventMonth" I can use regex and get the time_step parameters that way. Alternatively, if I use lower case parameters, the logic for the urlpatterns and the view method would be more complicated (a lot more if/else statements i.e. if "eventWeek": time_step = "weekly" and etc). I went for simplicity...lmk what you think